src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -12,6 +12,12 @@ namespace Microsoft.DotNet.Cli.Utils
     internal static class Constants
     {
         public static readonly string ExeSuffix = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? ".exe" : string.Empty;
+
+        // Priority order of runnable suffixes to look for and run
+        public static readonly string[] RunnableSuffixes = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                                                         ? new string[] { ".cmd", ".bat", ".exe" } 
+                                                         : new string[] { ".sh", string.Empty };
```

Comment:
We need to have one entry at least, the basic exe, to be backwards compatible with original code on Unix.